
# The name of the agv where the movable tray is located
string agv_name

# type of the movable tray
string movable_tray_type

# Name of the movable tray
string movable_tray_name

# Pose of the movable tray on the AGV
geometry_msgs/Pose pose